system/datatypes/peer_reviewed_article.php

```diff
@@ -1048,12 +1051,27 @@ private function getCommentFormHtml($type, DocPage $discussion = null) {
 		$comment = __('art-comment');
 		$new = $discussion === null;
 		$commentFieldName = self::FIELD_NAME_DISCUSSION_COMMENT . '/' . ($new ? 'new_' . $type : $discussion->getId());
-		$html = <<<EOF
+		if (strpos($commentFieldName, 'new_review') > 0) {
+			$_discussionLabel = __('art-start-discussion');
+			$_discussionShortTitle = __('art-title');
+
+			$html = <<<EOF
```

Comment:
This (and below) duplicates the entire html EOF block, so introduces quite some duplicate lines of (HTML) code. I wonder if it would be better to split the EOF block into separate lines instead, so you can always add the enclosing div tags and comment textarea, and make just the title stuff optional.